Makefile

```diff
@@ -83,9 +83,13 @@ e2e-ci-env-setup:
 	docker-compose run web bin/rails db:drop db:create db:schema:load && \
 	docker-compose run web bin/rails "cl2_back:create_tenant[e2e.front,e2etests_template]"
 
+e2e-ci-env-up:
```

Comment:
It is useful because code changes are applied only after the restart. And so we need an easy way to restart it without slow recreating of the DB.